x-pack/plugin/inference/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/xpack/application/CohereServiceUpgradeIT.java

```diff
@@ -48,14 +51,16 @@ public static void startWebServer() throws IOException {
         cohereRerankServer.start();
     }
 
-    // @AfterClass // for the awaitsfix
+    // TODO: disable again
+    @AfterClass // for the awaitsfix
     public static void shutdown() {
         cohereEmbeddingsServer.close();
         cohereRerankServer.close();
     }
 
     @SuppressWarnings("unchecked")
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/107887")
+    // TODO: mute again afterwards
+    // @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/107887")
```

Comment:
reminder: mute this test even, if it works again to keep re-enabling separate and isolated from this PR